File: qft-sim/src/physics/DecayMode.ts
import { KnownParticle } from './particles/ParticleTypes';

export class DecayMode {
    public readonly products: KnownParticle[];
    public readonly probability: number;

    constructor(products: KnownParticle[], probability: number) {
        this.products = products;
        this.probability = probability;
    }
}

File: SimGround/SimGround/Infra/Physics/DecayMode.cs
namespace SimGround.Infra.Physics
{
    using System.Collections.Generic;

    public class DecayMode
    {
        public readonly List<IParticle> products;
        public readonly double probability;

        public DecayMode(List<IParticle> products, double probability)
        {
            this.products = products;
            this.probability = probability;
        }
    }

}
